src/scantailor/filters/page_layout/alignment.py

```
@@ -0,0 +1,106 @@
+This module defines how pages are aligned both vertically and horizontally
+when their sizes differ.
+"""
+
+from __future__ import annotations
+
+from enum import Enum
+
+from pydantic import BaseModel
+
+
+class VerticalAlignment(str, Enum):
```

Ruff (0.15.0)

[warning] Class VerticalAlignment inherits from both `str` and `enum.Enum`

Inherit from `enum.StrEnum`

(UP042)